### PR TITLE
Added blockchain APIs

### PR DESCRIPTION
**Pull request by metachris**

### CODE REVIEW

**Comment by canesin** [comment]
missing ? https://neoscan.io/doc/Neoscan.Api.html

File: README.md [hunk 1]
```diff
@@ -55,15 +56,20 @@
 - [neotracker.io](https://neotracker.io/) - MainNet and TestNet explorer, created by the Neo Tracker team.
 - [explorer.neoverse.io](http://explorer.neoverse.io/) - Explorer created by the NEOverse team.
 
+## Blockchain APIs
+
+* [Neon wallet official REST api](https://github.com/CityOfZion/neon-wallet-db#overview-of-api)
+* [NEO API documentation](http://docs.neo.org/en-us/node/api.html) (possibly outdated)
```
Comment (revett) [inline README.md]: A-Z order of links please

File: README.md [hunk 2]
```diff
@@ -55,15 +56,20 @@
 - [neotracker.io](https://neotracker.io/) - MainNet and TestNet explorer, created by the Neo Tracker team.
 - [explorer.neoverse.io](http://explorer.neoverse.io/) - Explorer created by the NEOverse team.
 
+## Blockchain APIs
+
+* [Neon wallet official REST api](https://github.com/CityOfZion/neon-wallet-db#overview-of-api)
```
Comment (revett) [inline README.md]: Please add a short description, after the URI.

Comment (ixje) [reply README.md]: Please add in that description that the overview is partially outdated. It mixes `/v1/` and `/v2` api calls in the description and examples, while they all have a working `/v2/` version.

**Comment by metachris** [comment]
Closing this in favor of first collecting infos in #38

**Pull request closed without merging**